frontend/public/locales/en/apply-adult-child.json

```diff
@@ -124,8 +124,8 @@
           "not-eligible-purchased": "If a parent or legal guardian purchased the current dental insurance policy privately, a child is not eligible for the Canadian Dental Care Plan while that coverage is in effect.",
           "excepton": "Exception: a child may still be eligible if a parent or legal guardian retired and:",
           "exception-list": {
-            "opted-out": "Opted out of pension benefits before December 11, 2023, and",
-            "opt-back": "Can't opt back in under the pension rules"
+            "opted-out": "You opted out of pension benefits before December 11, 2023, and",
+            "opt-back": "You can't opt back in under the pension rules"
```

Comment:
You've changed the children's dental insurance content and what was there was good.

JSON path
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/855f0605-642e-440b-94b5-523d8a31d705)

Figma - adult-child - children - dental insurance

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/6ca455ae-74ff-4af9-9de6-1bac472bcb69)

Reply:
It's the same for the french content you changed.

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/d6b1ada2-328e-4d50-9db8-a13cf2726ae4)